meson.build

```diff
@@ -1616,6 +1616,14 @@ executable('systemd-debug-generator',
            install : true,
            install_dir : systemgeneratordir)
 
+executable('systemd-run-generator',
```

Comment:
The quoting in the man page is different than here in the commit message (it's wrong here IIUC). I think it's worth fixing to avoid confusion.